### ISSUE TITLE
Strangely inaccurate cuts

### ISSUE THREAD

**Issue by IlluminatiWave** [mifi/lossless-cut#2080]
### The fewer issues I have to read, the more new features I will have time to implement, so I ask that you please try these things first

- [X] Try with the [newest version from GitHub](https://github.com/mifi/lossless-cut/releases/latest) (also it might have already been fixed in latest [nightly build](https://github.com/mifi/lossless-cut#nightly-builds-))
- [X] Read the [documentation](https://github.com/mifi/lossless-cut) and [FAQ, Known issues, Troubleshooting](https://github.com/mifi/lossless-cut/blob/master/issues.md)
- [X] Search for your problem under [Issues](https://github.com/mifi/lossless-cut/issues) or [Discussions](https://github.com/mifi/lossless-cut/discussions)
- [X] If you have a problem only with a particular file (or file type), [use this form instead](https://github.com/mifi/lossless-cut/issues/new?assignees=&labels=&template=file-issue.yml)
- [X] Windows 7, 8 and 8.1 are [no longer supported](https://github.com/mifi/lossless-cut/discussions/1476). **This is not a question about Windows 8.1 or older.**

### Operating System

Windows 11

### Steps to reproduce

Cut segments where there are scene changes (actually it is in the vicinity of the keyframe as far as I can see).

### Expected behavior

Normal video cutting (actually there is no problem with a cut in the exact playable frame)

### Actual behavior

Normal cutting... BUT

Actually more than a problem, it is a doubt / bug.

I'm trying to cut a video with some scene changes, with the “keyframe cut mode” option disabled, I'm using cuts from frames, I'm making the cut at the exact frame of the scene change, it does it correctly, but there are usually cases where I can go back some frames, and I don't know if the videos save extra hidden information, but the thing is, I can go back between 10 and 30 frames and the scene change is simply not visible, but when playing, the change is noticeable.

For example, I want to split my video clip into different scenes where frames change, the first cut is at frame 600.

The first fragment is ok (0 ~ 600 frames)
making the second cut (601 ~ 900 frames) also seems right, but by fate (rather my OCD of having asked for 2 frames) made me realize that the second cut can be start from 571 and the scene would still be the same (even the frames 571 to 600 are coherent), but if I go down to 570, then all the initial frames change to the previous scene (the first 29 frames belong to the previous scene, which is coherent with the frames of the original video), then all the initial frames change to the previous scene (the first 29 frames belong to the previous scene, which is coherent with the frames of the original video, and it's strange because those 29 frames change according to the cutting order, although I guess it's more a video keyframe thing).

In summary, between frame 571 and 600 there seems to be a ghost zone that works as a patch for the initial / final part.

This is solved by activating smart cut, but my doubt is the reason for this behavior, and where do these extra frames come from? 

The video is an avc (h264) with a keyframe of 30 frames (which also agrees that the initial frames of each cut are *1, and down to *0 generate a curious / anomalous behavior, does that mean that between each keyframe there are ghost frames)?

**Comment by mifi** [comment]
this seems to be how ffmpeg works, unfortunately I haven't found out exactly how/why it works like this, but there has been attempts and discussions. see https://github.com/mifi/lossless-cut/blob/master/issues.md#cutting-times-are-not-accurate
